pf-recommended-practices/bp-guide-gh/ch1-model-formulation.md

```diff
@@ -185,11 +188,12 @@ and
 
 $${\bf J}_s=s^{NC}{\bf v} + \frac{1}{T}{\bf J}_e-\sum_i \left(\frac{\mu_i}{T}\right)^{NC}{\bf J}_i+\sum_i \alpha_i^2\frac{D\rho_i}{Dt}\nabla\rho_i+\epsilon^2\frac{D\phi}{Dt}\nabla\phi.$$
 
-It is worth noting that this form for ${\bf J}_s$ eliminates the explicit
-surface terms from the entropy production. We have also introduced the tensor
-${\bf Y}$ which is rather a complicated beast (it came from all those
-integrations by parts which we have skipped in this presentation), and has the
-form
+It is worth noting that this form for ${\bf J}_s$ eliminates the
+explicit surface terms from the entropy production. We have also
+introduced the tensor ${\bf Y}$ which is complicated beast <!-- which
```

Comment:
"is complicated beast" -> "is a complicated beast"

Reply:
got it